canvas_modules/common-canvas/src/common-properties/controls/slider/slider.scss

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017-2023 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ .properties-slider {
+    .bx--slider {
+        min-width: 8.6rem;
+    }
+    .bx--slider-text-input {
+        background-color: #F4F4F4;
```

Comment:
Why do you need to set this?  This should be controlled by `light` property on the control. It also needs to be set based on if it's in a tearsheet or right side flyout.